src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -120,8 +120,8 @@ def get_fuzz_task_payload(platform=None):
 
   query = data_types.FuzzerJobs.query()
   if environment.is_local_development():
-    query = query.filter(data_types.FuzzerJobs.platform.IN(platforms))[0]
-    mappings = list(ndb_utils.get_all_from_query(query))
+    query = query.filter(data_types.FuzzerJobs.platform.IN(platforms))
+    mappings = list(ndb_utils.get_all_from_query(query))[0]
```

Comment:
I think this is still wrong. Maybe do list(ndb_utils.get_all_from_query(query))[:1]

Reply:
Thanks for your proactivity here Jonathan. I expanded on the `fuzzer_selection_test : test_platform_restriction()` to account for this and it passes successfully, which lead me to believe this implementation worked correctly.

From a strictly pythonic standpoint, aren't the below two lines exactly the same (except in the case of no data found, which would lead to a `KeyError` in scenario 1)?

```
mappings = list(ndb_utils.get_all_from_query(query))[0]
vs.
mappings = list(ndb_utils.get_all_from_query(query))[:1]
```

Reply:
Regardless, I pushed the suggested change, but would highly prefer to verify it works so as not to break anything. This is a crucial snippet of code for bots.

Reply:
They arent the same. `L[0]` gets the 0th element in `L`, whereas `L[:1]` gets a list containing everything up to but not including the 1st element in L.
```
L = ['A', 'B']
L[0]   # 'A'
L[:1]  # ['A']
```